### DISCUSSION TITLE
Arduino Uno Wifi Rev2

### DISCUSSION THREAD

**Discussion by dtv42** [DCC-EX/CommandStation-EX#277] [General]
Hi,
I just found a reference to the Arduino Uno Wifi board in the code (DCCTimer.cpp):

#ifdef ARDUINO_ARCH_MEGAAVR
  // Arduino unoWifi Rev2 and nanoEvery architectire 

Does that imply I could use the Arduino Uno Wifi board for a CommandStation?

**Comment by habazut** [comment]
An UnoWifi does make Wifi happen very different from how it is done in DCC-EX with an Uno + ESP8266 Wifi. So even if the core part compiles, you don't have any Wifi yet. Then price and performance wise that board is not very intressting if you compare for example with an ESP32 in Uno form factor.

Harald.

**Comment by Asbelos** [comment]
This processor hardly has enough flash or ram  to include wifi and exrail processing and is staggeringly poor value for money.  I suggest it's a waste of resources to maintain any support for this board.On 9 Dec 2022 10:29, habazut ***@***.***> wrote:
An UnoWifi does make Wifi happen very different from how it is done in DCC-EX with an Uno + ESP8266 Wifi. So even if the core part compiles, you don't have any Wifi yet. Then price and performance wise that board is not very intressting if you compare for example with an ESP32 in Uno form factor.
Harald.
